docs/core/tools/dotnet-clean.md

```diff
@@ -4,29 +4,33 @@ description: The dotnet-clean command cleans the current directory.
 keywords: dotnet-clean, CLI, CLI command, .NET Core
 author: blackdwarf
 ms.author: mairaw
-ms.date: 03/06/2017
+ms.date: 03/09/2017
 ms.topic: article
 ms.prod: .net-core
 ms.technology: dotnet-cli
 ms.devlang: dotnet
 ms.assetid: eff65fa1-bab4-4421-8260-d0a284b690b2
 ---
-#dotnet-clean
+
+# dotnet-clean
 
 ## Name
 
 `dotnet-clean` - Cleans the output of a project. 
 
 ## Synopsis
 
-```
-dotnet clean [project] [-o|--output] [-f|--framework] [-c|--configuration] [-v|--verbosity]
-dotnet clean [--help] 
-```
+`dotnet clean [<PROJECT>] [-o|--output <OUTPUT_DIRECTORY>] [-f|--framework <FRAMEWORK>] [-c|--configuration <CONFIGURATION>] [-v|--verbosity <LEVEL>] [-h|--help]`
 
 ## Description
 
-The `dotnet clean` command will clean the output of the previous build. It is implemented as an MSBuild target, so the project will get evaluated. Only the outputs that were created during the build are cleaned. Both intermediate (`obj`) and final output (`bin`) folders are cleaned. 
+The `dotnet clean` command cleans the output of the previous build. It's implemented as an MSBuild target, so the project is evaluated when the command is run. Only the outputs created during the build are cleaned. Both intermediate (`obj`) and final output (`bin`) folders are cleaned.
```

Comment:
MSBuild target -> link to https://docs.microsoft.com/visualstudio/msbuild/msbuild-targets?